nerfstudio/viewer/app/src/modules/RenderModal/RenderModal.jsx

```diff
@@ -19,14 +19,25 @@ export default function RenderModal(props: RenderModalProps) {
     (state) => state.renderingState.config_base_dir,
   );
 
+  const timestamp_regex = /[0-9]{4}-[0-9]{2}-[0-9]{2}_[0-9]{6}/g;
```

Comment:
Is it alright if I get the timestamp of the directory like this? I figured it was simpler than sending the timestamp in a websocket but that might be cleaner?
I'm happy to change this
@ethanweber